### DISCUSSION TITLE
Stable version of build123d?

### DISCUSSION THREAD

**Discussion by ysavtchouk-pf** [gumyr/build123d#412] [Q&A]
Hi all! First, thank you so much for the awesome work you are doing!
We are considering using build123d as CAD tool, but we currently run into installation problems. I was wondering, is there already a stable version that we can try out?

I see that the default branch is 'dev' and the 'Wheel building and publishing' action is currently failing?

**Comment by jdegenstein** [comment]
Hello and welcome to build123d! 

build123d has not had an official stable release yet (it will be 1.0.0). See here for the remaining issues before this milestone: https://github.com/gumyr/build123d/milestone/1

In the meantime you can install per the instructions here: https://build123d.readthedocs.io/en/latest/installation.html

Let me know if you need further assistance with installation.
